src/stream.cc

```diff
@@ -24,140 +24,110 @@
 
 namespace wabt {
 
-static FileStream s_stdout_stream;
-static FileStream s_stderr_stream;
-
-void init_stream(Stream* stream, Writer* writer, Stream* log_stream) {
-  stream->writer = writer;
-  stream->offset = 0;
-  stream->result = Result::Ok;
-  stream->log_stream = log_stream;
+Stream::Stream(Writer* writer, Stream* log_stream)
+    : writer_(writer),
```

Comment:
Oh my, the trailing underscore naming convention is so ugly, no?  I don't understand why anyone would choose this...but I guess its our chosen style for chrome?